runtime/src/pallets_governance.rs

```diff
@@ -0,0 +1,184 @@
+/*
+ * This file is part of the Nodle Chain distributed at https://github.com/NodleCode/chain
+ * Copyright (C) 2020  Nodle International
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use crate::{
+    constants, Call, Event, FinancialCommittee, Origin, OriginCaller, RootCommittee, Runtime,
+    Scheduler, TechnicalCommittee,
+};
+use frame_support::parameter_types;
+use nodle_chain_primitives::{AccountId, BlockNumber};
+use sp_core::u32_trait::{_1, _2};
+use sp_runtime::ModuleId;
+pub use sp_runtime::{Perbill, Perquintill};
+
+// Shared parameters with all collectives / committees
+parameter_types! {
+    pub const MotionDuration: BlockNumber = 2 * constants::DAYS;
+    pub const MaxProposals: u32 = 100;
+    pub const MaxMembers: u32 = 50;
+}
+
+pub type TechnicalCollective = pallet_collective::Instance2;
+impl pallet_collective::Config<TechnicalCollective> for Runtime {
+    type Origin = Origin;
+    type Proposal = Call;
+    type Event = Event;
+    type MotionDuration = MotionDuration;
+    type MaxProposals = MaxProposals;
+    type WeightInfo = ();
+    type MaxMembers = MaxMembers;
+    type DefaultVote = pallet_collective::PrimeDefaultVote;
+}
```

Comment:
Let's move this down to the `Technical committee` section

Reply:
Thanks Eliott

I hope now I aligned & paired it properly

| committee | pallet_collective | pallet_membership |
| --- | --- | --- |
| Technical | Instance2 | Instance1 |
| Financial | Instance3 | Instance3 |
| Root | Instance4 | Instance4 |